### ISSUE TITLE
PSO2 No longer recognizing PhysXCooking_64.dll, PigPSOCamNGS.dll flagged as Trojan by Windows.

### ISSUE THREAD

**Issue by sorasin** [Lapig/PigPSO2Cam#3]
Upon launching PSO2 it cannot find PhysXCooking, and Windows is now reading PigPSOCam .dll as a virus immediately after trying to boot.

Uninstalling the mod by replacing PhysXCooking_64.dll with backup restores normal game function.

See image for virus flag.

![pigpsoflag](https://user-images.githubusercontent.com/74996668/125330289-9135b800-e314-11eb-9036-904aa6fd0059.PNG)

Thanks

**Comment by Lapig** [comment]
[ScarletFlash.A MSDN Entry](https://www.microsoft.com/en-us/wdsi/threats/malware-encyclopedia-description?Name=Virus:Win32/ScarletFlash.A), pigcam gets loaded automatically by adding an import for the dll onto physxcooking, when pigcam is in quarantine and cannot be loaded you will get that error message trying to start the game. The program does execute commands from an "attacker" (me) and those commands are what change the fov and zoom

tldr False positive, just restore and allow the file
